redis-bom/build.gradle

```diff
@@ -1,8 +1,3 @@
 plugins {
     id "io.micronaut.build.internal.bom"
 }
-dependencies {
-    constraints {
-        api("io.lettuce:lettuce-core:$lettuceVersion")
```

Comment:
why is this no longer there? Is it included automatically because it is specified in `lib.versions.toml`?

Reply:
Yes, Cédric's [comment here](https://github.com/micronaut-projects/micronaut-redis/pull/268#issuecomment-1064127959) is what led to this PR

Reply:
but with this change if I understand correctly we are going to expose `redis-embedded` in the bom. Which I think we should not since I believe is test dependency.

Reply:
No, because it's not `managed-`

Having the managed prefix here

```
managed-lettuce = { module = "io.lettuce:lettuce-core", version.ref = "managed-lettuce" }
```

Means it ends up in the toml:

```toml
[versions]
lettuce = "6.1.6.RELEASE"

[libraries]
lettuce = {group = "io.lettuce", name = "lettuce-core", version.ref = "lettuce" }
```

And the POM:

```xml
      <dependency>
        <groupId>io.lettuce</groupId>
        <artifactId>lettuce-core</artifactId>
        <version>${lettuce.version}</version>
      </dependency>
```

@melix can you confirm if I'm right?

Reply:
Yes, that's correct.

Reply:
[The wiki says](https://github.com/micronaut-projects/micronaut-build/wiki/Upgrade-guide): 

> Note that if your project uses a version catalog, then this plugin can generate a BOM in a similar way to what the micronaut-core project does (with the convention for managed dependencies).`

It would be worth clarifying that `managed` convention there.